src/App.js

```diff
@@ -6,7 +6,7 @@ class App extends Component {
   render() {
     return (
       <div>
-        <div>Welcome to trello board</div>
+        <div style={{textAlign:"center"}}>Welcome to trello board</div>
```

Comment:
Use proper spacing.
It should be like this `<div style={{ textAlign: 'center' }}>Welcome to trello board</div>`